source/quick-start-sinatra/download-and-install.txt

```diff
@@ -0,0 +1,111 @@
+.. _mongoid-qs-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: ruby framework, odm, code example
+
+Prerequisites
+-------------
+
+To create the Quick Start application by using Sinatra, you need the
+following software installed in your development environment:
+
+- `Ruby <https://www.ruby-lang.org/en/documentation/installation/>`__
+- `RubyGems package manager <https://rubygems.org/pages/download>`__
+- A terminal app and shell. For MacOS users, use Terminal or a similar app.
+  For Windows users, use PowerShell.
+
+Download and Install the Mongoid Gem
```

Comment:
S: You could use the {+odm+} constant here (I know that makes the heading dashes weird